cmd/kosli/snapshotPaths.go

```diff
@@ -3,10 +3,12 @@ package main
 import (
 	"fmt"
 	"io"
+	"log"
 	"net/http"
 	"path/filepath"
 	"strings"
 
+	"github.com/fsnotify/fsnotify"
```

Comment:
Be aware that fsnotify does not work for subdirectories:

> Are subdirectories watched?
No, you must add watches for any directory you want to watch (a recursive watcher is on the roadmap: https://github.com/fsnotify/fsnotify/issues/18).